workflow/controller/dag.go

```diff
@@ -233,12 +233,32 @@ func (woc *wfOperationCtx) executeDAG(nodeName string, tmplCtx *templateresoluti
 	for _, taskNames := range targetTasks {
 		woc.executeDAGTask(dagCtx, taskNames)
 	}
+
+	updateOutboundNodes := func() {
```

Comment:
Is it possible to define this as a method like "steps" does?
https://github.com/argoproj/argo/blob/9035286554768119b83e00294ea7c0b400b5de83/workflow/controller/steps.go#L142

In the future, we may want to refactor these template handling, then using similar structure wii make it easier.

What do you think?

Reply:
done